kolibri/core/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "kolibri-core",
```

Comment:
In the notes you say

> Creates a `kolibri` npm package that creates an (somewhat complete, but with extant issues for some use cases) exported copy of the Kolibri Core API currently only intended for use in tests of external plugins (as any imports of the core API will need to be properly resolved for the purposes of executing tests).

I'm a little unclear what this is and how it works.

As an aside, wouldn't the package be `kolibri-core` rather than `kolibri`?

Reply:
No, the `kolibri` package is constructed with reference to this `kolibri-core` package, but is a separate entity. The main purpose of calling the exported package `kolibri` is for consistency of naming between our internal references to the kolibri core API, and people using it externally.